src/TTextEdit.cpp

```diff
@@ -1784,6 +1805,433 @@ int TTextEdit::getRowCount()
+                        .arg(tr("Index (UTF-16)", "1st Row heading for Text analyser output, table item is the count into the QChars/TChars that make up the text {this translation used 2 times}"), indexes)
+                        .arg(tr("U+<i>####</i> Unicode Code-point <i>(High:Low Surrogates)</i>", "2nd Row heading for Text analyser output, table item is the unicode code point (will be between 000001 and 10FFFF in hexadecimal) {this translation used 2 times}"), vals)
+                        .arg(tr("Visual", "3rd Row heading for Text analyser output, table item is a visual representation of the character/part of the character or a '{'...'}' wrapped letter code if the character is whitespace or otherwise unshowable {this translation used 2 times}"), elements)
+                        .arg(tr("Index (UTF-8)", "4th Row heading for Text analyser output, table item is the count into the bytes that make up the UTF-8 form of the text that the Lua system uses {this translation used 2 times}"), utf8Indexes)
+                        .arg(tr("Byte", "5th Row heading for Text analyser output, table item is the unsigned 8-bit integer for the particular byte in the UTF-8 form of the text that the Lua system uses {this translation used 2 times}"), utf8Vals)
+                        .arg(tr("Lua character or code", "6th Row heading for Text analyser output, table item is either the ASCII character or the numeric code for the byte in the row about this item in the table, as displayed the thing shown can be used in a Lua string entry to reproduce this byte {this translation used 2 times}"), luaCodes);
+                isFirstRow = false;
+            } else {
+                completedRows.append(QStringLiteral("<small><table border=\"1\" style=\"margin-top:5px; margin-bottom:5px; margin-left:5px; margin-right:5px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">"
+                                                    "<tr>%1</tr>"
+                                                    "<tr>%2</tr>"
+                                                    "<tr>%3</tr>"
+                                                    "<tr>%4</tr>"
+                                                    "<tr>%5</tr>"
+                                                    "<tr>%6</tr>"
+                                                    "</table></small><br>")
+                                     .arg(indexes, vals, elements, utf8Indexes, utf8Vals, luaCodes));
+            }
+            rowItems = 0;
+            indexes.clear();
+            vals.clear();
+            elements.clear();
+            utf8Indexes.clear();
+            utf8Vals.clear();
+            luaCodes.clear();
+        }
+    }
+
+    if (mpContextMenuAnalyser) {
+        if (isFirstRow) {
+            // if this is still true then we only have a short, single line of
+            // less than 16 codepoints
+            mpContextMenuAnalyser->setToolTip(QStringLiteral("<html><head/><body>%1"
```

Comment:
I'm still having difficulty accepting the practicality of a tooltip for this. If I want to copy the Lua `\###` out into code, I can't. If I want to copy the unicode codepoint for Googling, I can't. I also always click on the analyser first because that's how right-click menus always work - you click on them, looking at a tooltip goes against what I've learnt. :(

Reply:
I never intended to make it possible to copy from the table - I just wanted it to be a quick pop-up of something informational and then to go away on its own.

I am willing to consider revising the UI aspect of it so that it becomes more of a *tear-off* (if that is the right term) - would you be willing to take the UI as it is and then get that revised in a later PR?

Reply:
I might then be in a better position then to also squeeze in something about the colouring - or provide a second but separate way to see all the SGR effects with a similar revised interface...